src/m365/spo/commands/site/site-recyclebinitem-restore.spec.ts

```diff
@@ -96,6 +97,24 @@ describe(commands.SITE_RECYCLEBINITEM_RESTORE, () => {
     assert.strictEqual(actual, true);
   });
 
+  it('validates for a correct input with ids', async () => {
```

Comment:
Isn't this the same test as `passes validation when multiple IDs are specified`?